uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget.component.js

```diff
@@ -1,13 +1,29 @@
 /* eslint-env amd */
 
 define([
+  'common/angular',
   'common/lodash',
+  'common/bundles/components',
+  'common/modules/directives',
+  'common/modules/models',
```

Comment:
wait, this can't be right

1. You should not require the whole bundle in any circumstance. The bundle is used only for building reqangular.min.js, nothing else
2. You should not require the modules of `common/` either. You should just require what you actually need (a service, a component, a directive, etc) which in turn will require internally its own module

Reply:
I think this part was giving me issues, I'll give it a try and let you know what the specific issue was (if any)

Reply:
if it was, then something was definitely wrong. The above should not be like that, no exception